app/frontend/src/styles/application/home.scss

```diff
@@ -0,0 +1,72 @@
+.app-search-form-container {
+  padding: govuk-spacing(4);
+  color: #fff;
+  background: url('../../images/TVS-logo.png') no-repeat 100% 10px/120px auto govuk-colour("dark-blue");
```

Comment:
Does this play nicely with the asset pipeline?

Reply:
Yes, it renders the image as you can see in the review app.